shopify/customized-bundle-template/web/components/BundleForm.tsx

```diff
@@ -15,13 +20,112 @@ import { useCallback, useContext, useEffect, useState } from "react";
 import { ShopContext } from "../providers";
 import { useAppBridge } from "@shopify/app-bridge-react";
 import ProductCard from "./ProductCard";
+import { ShopContextType } from "../providers/ShopProvider";
+
+export type Variant = {
```

Comment:
Is this just our product variant type? We should have that available without having to redefine it

Reply:
No, its the type that Shopify sends back in the resource picker